_posts/2019-05-08-utah-sgid-statewide-roads-data-layer-updates-05082019.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Utah SGID Statewide Roads Data Layer Updates 05/08/2019
+author:
+  display_name: Greg Bunce
+  email: gbunce@utah.gov
+date: 2019-05-08 10:27:06
+categories:
+  - Featured
+tags:
+  - roads
+---
+
+AGRC recently completed updates to the SGID10.Transportation.Roads feature class that resides on the [Utah SGID ArcSDE database server]({{ "/sgid-database/" | prepend: site.baseurl }}). Updated roads data is also available as shapefiles and geodatabase files for download on the [SGID Roads and Highway System download page]({{ "/data/transportation/roads-system/" | prepend: site.baseurl }}). These updates are also reflected in [AGRC's address locators]({{ "/data/address-geocoders-locators/" | prepend: site.baseurl }}) and the [Utah Next-Generation 911 GIS Database]({{ "/data/911/" | prepend: site.baseurl }}).
+
+
+The following are highlights from this month's update.
+
+#### County Updates
+
+New roads were added and road names and address ranges were updated for the following counties:
+
+- **Box Elder County:** Obtained roads data on 03/07/2019. Previous update was on 02/23/2019.
+
```

Comment:
Line feeds are not necessary when using bullets

Reply:
got it.  thx